foreman/foreman.spec

```diff
@@ -570,6 +588,8 @@ rm -rf %{buildroot}
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
 %config %{_sysconfdir}/cron.d/%{name}
 %{_sysconfdir}/rpm/macros.%{name}
+%{_sysconfdir}/rpm/macros.%{name}-apipie
+%{_sysconfdir}/rpm/macros.%{name}-assets
```

Comment:
Sorry, I meant that if you're adding the -assets macro file to 'foreman' then it can be removed from 'foreman-assets' so it's not in both (though harmless).

I do kind of like the idea of a -plugin or -build subpackage though, which could depend on foreman-sqlite and ship these macro files.  Or we repurpose foreman-assets for this too?